Mathlib/Analysis/Normed/Group/SemiNormedGroup.lean

```diff
@@ -0,0 +1,300 @@
+/-
+Copyright (c) 2021 Johan Commelin. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Johan Commelin, Riccardo Brasca
+
+! This file was ported from Lean 3 source module analysis.normed.group.SemiNormedGroup
+! leanprover-community/mathlib commit 17ef379e997badd73e5eabb4d38f11919ab3c4b3
+! Please do not edit these lines, except to modify the commit id
+! if you have ported upstream changes.
+-/
+import Mathlib.Analysis.Normed.Group.Hom
+import Mathlib.CategoryTheory.Limits.Shapes.ZeroMorphisms
+import Mathlib.CategoryTheory.ConcreteCategory.BundledHom
+import Mathlib.CategoryTheory.Elementwise
+
+/-!
+# The category of seminormed groups
+
+We define `SemiNormedGroup`, the category of seminormed groups and normed group homs between them,
+as well as `SemiNormedGroup₁`, the subcategory of norm non-increasing morphisms.
+-/
+
+
+noncomputable section
+
+universe u
+
+open CategoryTheory
+
+/-- The category of seminormed abelian groups and bounded group homomorphisms. -/
+def SemiNormedGroup : Type (u + 1) :=
```

Comment:
I don't think we want to keep this name, it's way too similar to `SeminormedGroup` which means something quite different! Is the convention to add a `Cat` suffix?

Reply:
(the same comment applies to the filename)

Reply:
Thanks, done! (Also renamed SemiNormedGroup_1 correspondingly)